### PR TITLE
Showing Snackbar instead of Toast in Reviewer.kt

### PR DESCRIPTION
**Pull request by SanjaySargam**
## Purpose / Description
Used Snackbar instead of Toast

## Fixes
Related #13411

## How Has This Been Tested?
Tested on Emulator

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt [hunk 1]
```diff
@@ -149,10 +149,8 @@ open class Reviewer :
             // BUG: If the method crashes, this will crash
             invalidateOptionsMenu()
             val cardCount: Int = result!!.value.result.size
-            showThemedToast(
-                this,
-                resources.getQuantityString(toastResourceId, cardCount, cardCount),
-                true
+            this.showSnackbar(
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt:152]: ditto

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt [hunk 2]
```diff
@@ -445,10 +443,10 @@ open class Reviewer :
                 if (whiteboard != null) {
                     try {
                         val savedWhiteboardFileName = whiteboard!!.saveWhiteboard(TimeManager.time).path
-                        showThemedToast(this@Reviewer, getString(R.string.white_board_image_saved, savedWhiteboardFileName), true)
+                        this.showSnackbar(getString(R.string.white_board_image_saved, savedWhiteboardFileName))
                     } catch (e: Exception) {
                         Timber.w(e)
-                        showThemedToast(this@Reviewer, getString(R.string.white_board_image_save_failed, e.localizedMessage), true)
+                        this.showSnackbar(getString(R.string.white_board_image_save_failed, e.localizedMessage))
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt:449]: ditto

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt [hunk 3]
```diff
@@ -149,10 +149,8 @@ open class Reviewer :
             // BUG: If the method crashes, this will crash
             invalidateOptionsMenu()
             val cardCount: Int = result!!.value.result.size
-            showThemedToast(
-                this,
-                resources.getQuantityString(toastResourceId, cardCount, cardCount),
-                true
+            this.showSnackbar(
+                resources.getQuantityString(toastResourceId, cardCount, cardCount)
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt:153]: previously the toast time was short and the snackbar is long

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt (hunk 2 above)
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt:449]: ditto 2 (snackbar time)

**Review by BrayanDSO — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt [hunk 4]
```diff
@@ -693,7 +693,7 @@ open class Reviewer :
     @NeedsTest("Starting animation from swipe is inverse to the finishing one")
     protected fun openCardInfo(fromGesture: Gesture? = null) {
         if (currentCard == null) {
-            showThemedToast(this, getString(R.string.multimedia_editor_something_wrong), true)
+            showSnackbar(getString(R.string.multimedia_editor_something_wrong))
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt:696]: This also requires `Snackbar.LENGTH_SHORT`.

**Review by lukstbit — CHANGES_REQUESTED** [review]

**Review by lukstbit — APPROVED** [review]

**Pull request merged**